xmodule/tests/__init__.py

```diff
@@ -159,6 +160,7 @@ def get_block(block):
 
     services = {
         'user': user_service,
+        'deprecated_anonymous_id': DeprecatedAnonymousIdService('student'),
```

Comment:
it's probably worth noting that anonymous_ids basically have to be hex strings at this point, because that's what they've always been, and there's content out there that depends on this for randomization behavior. 😞